docs/rfc/0696-p2p-node.adoc

```diff
@@ -0,0 +1,318 @@
+= RFC: Peer-to-peer Node
+:author: @FintanH, @xla
+:revdate: 2021-06-31
+:revremark: draft
+:toc:
+:toc-placement: preamble
+
+* Author: {author}
+* Date: {revdate}
+* Status: {revremark}
+* Community discussion: n/a
+* Tracking Issue: n/a
+
+== Motivation
+
+So far long-running binaries providing a link p2p daemon came in two
+incarnations, the *radicle-upstream* proxy and the *radicle-bins* seed node.
+With the former being the main subject of experimentation of higher-level
+non-core protocols (e.g. requesting new identities on the network) it developed
+into a monolith due to its locality close to the upstream project and a constant
+push to reflect highly specific product needs to drive the UI. While beneficial
+to validate the feasibility of the general ideas of the link protocol it now
+presents the reality that there is no simple way to run the link p2p stack
+without the idiosyncrasies of the application specific proxy. This is a massive
+hindrance for further adoption and recomposition in other contexts and needs to
+be addressed to enable the development of a healthy ecosystem.
+
+== Overview
+
+This RFC expands on the peer-to-peer node outlined in RFC 682 to propose a new
+stand-alone daemon with focus on driving the p2p stack, exposing a minimal API
+and defer supervision to more integrated and mature system services. Furthermore
+it tries to prove that all daemons are made equal and only distinct with regards
+to the configuration given (e.g. tracking behaviour) and lifetime of the process
+(i.e. socket-activated or long-running).
+
+It also strives to recommend APIs and mechanisms - in the form of non-core
+protocols - which remove the need for online requests made by clients to a
+running node.
```

Comment:
I think this would perhaps do as `Motivation` :slightly_smiling_face:

Reply:
Are you referring to the entire section, or just the last paragraph?

Reply:
I'm suggesting to remove `Motivation`, but keep `Overview`. #682 already used sufficiently dismissive language to motivate the move ^)

Reply:
With a heavy heart shall I bid goodbye to my beloved prose.

Reply:
7674dcf

Reply:
7674dcf